src/components/Footer.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+import "../App.css";
```

Comment:
I would move this to App.js so you don't have to import it on all the files.

Reply:
thank you! made this change